internal/collectors/virtualization.go

```diff
@@ -0,0 +1,34 @@
+package collectors
+
+import (
+	"fmt"
+
+	"github.com/SUSE/connect-ng/internal/util"
+)
+
+type Virtualization struct{}
+
+func (Virtualization) run(arch Architecture) (Result, error) {
+
+	// We utilize systemd here to fetch the virtualization information. Since we do not hard require systemd
+	// there might be the possibility to run on a system without systemd installed (e.g. containers).
```

Comment:
On that regard, could we consider detecting container engines as well? (see [this package](https://github.com/genuinetools/bpfd/tree/master/proc) which is being used by tools like [amicontained](https://github.com/genuinetools/amicontained).

Reply:
We have this in our next epic CAT see our Roadmap https://trello.com/c/v0ae4HG6/450-cat-collect-additional-telemetry-data